lib/ansible/modules/cloud/google/gcp_spanner_instance.py

```diff
@@ -307,6 +308,41 @@ def response_to_hash(module, response):
     }
 
 
+def async_op_url(module, extra_data=None):
+    if extra_data is None:
+        extra_data = {}
+    url = "https://spanner.googleapis.com/v1/projects/{project}/global/operations/{op_id}"
```

Comment:
This is wrong, of course - but it's not obviously more wrong than the existing behavior, which doesn't wait at all.  What should I do?

Reply:
Merge it in and file a bug.